README.md

```diff
@@ -1 +1,91 @@
-# sensuctl-plugin-template
\ No newline at end of file
+# Sensuctl Plugin Template
+
+## Overview
+sensuctl-plugin-template is a template repository which wraps the [Sensu Plugin SDK][2].
+To use this project as a template, click the "Use this template" button from the main project page.
+Once the repository is created from this template, you can use the [Sensu Plugin Tool][9] to
+populate the templated fields with the proper values.
+
+## Functionality
+
+After successfully creating a project from this template, update [main.go][7] to customize the
+behavior of this sensuctl plugin.
+
+When writing or updating a plugin's README from this template, review the Sensu Community
+[plugin README style guide][3] for content suggestions and guidance. Remove everything
+prior to `# {{ .Name }}` from the generated README file, and add additional context about the
+plugin per the style guide.
+
+## Releases with Github Actions
+
+To release a version of your project, simply tag the target sha with a semver release without a `v`
+prefix (ex. `1.0.0`). This will trigger the [GitHub action][5] workflow to [build and release][4]
+the plugin with goreleaser. Register the asset with [Bonsai][8] to share it with the community!
+
+***
+
+# {{ .Name }}
+
+## Table of Contents
+- [Overview](#overview)
+- [Files](#files)
+- [Usage examples](#usage-examples)
+- [Configuration](#configuration)
+  - [Asset registration](#asset-registration)
+  - [Resource definition](#resource-definition)
+- [Installation from source](#installation-from-source)
+- [Additional notes](#additional-notes)
+- [Contributing](#contributing)
+
+## Overview
+
+The {{ .Name }} is a [Sensu CLI][6] tool (`sensuctl`) that ...
+
+## Files
+
+## Usage examples
+
+## Configuration
+
+### Asset registration
+
+[Sensu Assets][10] are the best way to make use of this plugin. If you're not using an asset, please
+consider doing so! If you're using sensuctl 5.13 with Sensu Backend 5.13 or later, you can use the
+following command to add the asset:
+
+```
+sensuctl asset add {{ .GithubUser }}/{{ .GithubProject }}
+```
+
+If you're using an earlier version of sensuctl, you can find the asset on the [Bonsai Asset Index][https://bonsai.sensu.io/assets/{{ .GithubUser }}/{{ .GithubProject }}].
+
+### Resource definition
+
+## Installation from source
+
+The preferred way of installing and deploying this plugin is to use it as an Asset. If you would
+like to compile and install the plugin from source or contribute to it, download the latest version
+or create an executable script from this source.
+
+From the local path of the {{ .GithubProject }} repository:
+
+```
+go build -o /usr/local/bin/{{ .GithubProject }} main.go
```

Comment:
We shouldn't output to a system directory like that: that forces `go build` to run as `root`, which it should never do. `go build` only builds, it doesn't install! People need to run `go install` if they want to "install" the resulting binary.

Also `go help build` mentions:
> If the arguments to build are a list of .go files from a single directory, build treats them as a list of source files specifying a single package.

I wonder if specifying `main.go` breaks the build if the `main` package is split across several files (I haven't tried).

I think we should replace that line with just `go build`, or _maybe_ `go build -o {{ .GithubProject }}` if we really want that particular name for the binary.

Reply:
Following the instructions from https://github.com/sensu-plugins/community/blob/master/PLUGIN_STYLEGUIDE.md#installation-from-source. If those instructions are incorrect, we should start there.

Reply:
I think we should nip this in the bud before people start (hopefully) using those templates, but I don't want to block this PR.

Here's the course of action I suggest: I'll open issues against the plugin style guide and the existing plugins that it references. If people agree with my proposed changes, I can then open appropriate PRs across the board.